File: Semester3/MAP/Seminare/Seminar14/src/main/java/org/example/seminar14/Domain/Angajat.java
package org.example.seminar14.Domain;

public class Angajat {
    private int id;
    private String name;
    private double venitPeOra;
    private Nivel senioritate;

    public Angajat(String name, double venitPeOra, Nivel senioritate) {
        this.name = name;
        this.venitPeOra = venitPeOra;
        this.senioritate = senioritate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getVenitPeOra() {
        return venitPeOra;
    }

    public void setVenitPeOra(double venitPeOra) {
        this.venitPeOra = venitPeOra;
    }

    public Nivel getSenioritate() {
        return senioritate;
    }

    public void setSenioritate(Nivel senioritate) {
        this.senioritate = senioritate;
    }
}

File: Semester3/MAP/Seminare/Sem11/Sem11/domain/Entitys/Angajat.cs
namespace Sem11;

public class Angajat : Entity<String>
{
    private string nume;
    private double venitPeOra;
    private Nivel nivel;

    public Angajat(string nume, double venitPeOra, Nivel nivel)
    {
        this.nume = nume;
        this.venitPeOra = venitPeOra;
        this.nivel = nivel;
    }
    
    public string Nume { get => nume; set => nume = value; }
    
    public double VenitPeOra { get => venitPeOra; set => venitPeOra = value; }
    
    public Nivel Nivel { get => nivel; set => nivel = value; }

    public override string ToString()
    {
        return $"{nume}: {venitPeOra} : {nivel}";
    }
}